packages/ethereum-types/src/consensus/light_client_header.rs

```diff
@@ -1,144 +1,132 @@
-//! This module defines the types used in the light client updates
+//! This module defines types related to beacon's light client functionality.
 
-use alloy_primitives::{Address, B256, U256};
+use alloy_primitives::{Address, Bloom, Bytes, B256, U256};
+use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 use tree_hash_derive::TreeHash;
 
-use crate::config::consts::{
-    EXECUTION_BRANCH_DEPTH, FINALITY_BRANCH_DEPTH, NEXT_SYNC_COMMITTEE_BRANCH_DEPTH,
-};
-
 use super::{
-    sync_committee::{SyncAggregate, SyncCommittee, TrustedSyncCommittee},
-    wrappers::{WrappedBloom, WrappedBranch, WrappedBytes},
+    merkle::{EXECUTION_BRANCH_DEPTH, FINALITY_BRANCH_DEPTH, NEXT_SYNC_COMMITTEE_BRANCH_DEPTH},
+    sync_committee::{SyncAggregate, SyncCommittee},
 };
 
-/// The header of a light client update
-#[derive(Serialize, Deserialize, PartialEq, Eq, Clone, Debug, Default)]
-pub struct Header {
-    /// The trusted sync committee
-    pub trusted_sync_committee: TrustedSyncCommittee,
-    /// The consensus update
-    pub consensus_update: LightClientUpdate,
-    /// The account update
-    pub account_update: AccountUpdate,
-}
-
 /// A light client update
-#[derive(Serialize, Deserialize, PartialEq, Eq, Clone, Debug, Default)]
+#[derive(Serialize, Deserialize, JsonSchema, PartialEq, Eq, Clone, Debug, Default)]
 #[allow(clippy::module_name_repetitions)]
 pub struct LightClientUpdate {
     /// Header attested to by the sync committee
     pub attested_header: LightClientHeader,
     /// Next sync committee corresponding to `attested_header.state_root`
-    #[serde(default)] // TODO: Check if this can be removed in #143
     pub next_sync_committee: Option<SyncCommittee>,
     /// The branch of the next sync committee
-    pub next_sync_committee_branch: Option<WrappedBranch<NEXT_SYNC_COMMITTEE_BRANCH_DEPTH>>,
+    #[schemars(with = "Vec<String>")]
+    pub next_sync_committee_branch: Option<[B256; NEXT_SYNC_COMMITTEE_BRANCH_DEPTH]>,
     /// Finalized header corresponding to `attested_header.state_root`
     pub finalized_header: LightClientHeader,
     /// Branch of the finalized header
-    pub finality_branch: WrappedBranch<FINALITY_BRANCH_DEPTH>,
+    #[schemars(with = "Vec<String>")]
+    pub finality_branch: [B256; FINALITY_BRANCH_DEPTH],
     /// Sync committee aggregate signature
     pub sync_aggregate: SyncAggregate,
     /// Slot at which the aggregate signature was created (untrusted)
+    #[serde(with = "utils::serde::number_as_string")]
```

Comment:
I actually did this on purpose to avoid having to add serde_with as a dependency for a single type. I am also OK with adding it, but just seems unnecessary :)

Reply:
I think you use this in other places as well

Reply:
Not serde_with. This particular serde module, yes, but it is the only thing.

Reply:
I think introducing this dep is better than maintaining a function in our codebase